src/Model/DICOMDirectorySearch.py

```diff
@@ -20,20 +20,22 @@ def get_dicom_structure(path, interrupt_flag, progress_callback):
     dicom_structure = DICOMStructure()
 
     files_searched = 0
-    total_files = sum([len(files) for root, dirs, files in os.walk(path)])
 
     no_patient_id = 1
```

Comment:
This is a count of the number of files lacking a patient ID.  A variable renaming would make the code involving this easier to read (and review, and maintain)